playlists/excel.yaml

```diff
@@ -9,6 +9,7 @@
   group: Basics
   api_set:
     ExcelApi: 1.1
+  type: public
 - id: excel-basic-api-call-es5
   name: Basic API call (JavaScript)
   fileName: basic-api-call-es5.yaml
```

Comment:
Are we ever using `filename` and `relativepath` in Script Lab?  If not, shall we get rid of it to save bandwidth?  Ditto for the `all` playlist.